Src/Web/Web.Shared.Net/RequestTrackingTelemetryModule.cs

```diff
@@ -402,14 +402,16 @@ private class ChildRequestTrackingSuppressionModule
             /// <summary>
             /// Max number of request ids to cache.
             /// </summary>
-            private const int MAXSIZE = 100;
+            private const int MAXSIZE = 10000;
 
+            private const int TIMEOUTSECONDS = -3;
+            
             private const string RequestIdHeader = "ApplicationInsights-RequestTrackingTelemetryModule-Request-Id";
 
             /// <summary>
-            /// Using this as a hash-set of current active requests. The second value is ignored.
+            /// Using this as a hash-set of current active requests. The second value is used when auditing the dictionary for cleanup.
             /// </summary>
-            private static ConcurrentDictionary<string, byte> activeRequests = new ConcurrentDictionary<string, byte>();
+            private static ConcurrentDictionary<string, long> activeRequests = new ConcurrentDictionary<string, long>(System.Environment.ProcessorCount, MAXSIZE);
```

Comment:
is there Risk of an exception being thrown when trying to read System. ? If yes this code is not safe. Better to do this inside try catch.